sos/plugins/container_log.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (C) 2019 Red Hat, Inc., Cedric Jeanneret <cjeanner@redhat.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.plugins import Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin
+import os
+
+
+class ContainerLog(Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin):
+    '''All container logs
+    '''
+    plugin_name = 'container_log'
+    files = ('/var/log/containers',)
+
+    def setup(self):
+        if self.get_option('all_logs'):
+            self.add_copy_spec([
+                '/var/log/containers/',
+            ])
+        else:
+            self.add_copy_spec(self._subdirs('/var/log/containers'))
```

Comment:
Checking the merge conflict in `nova`:

1) it will be OK to replace `for novadir in novadirs:` loop by simple replacing `novadir` by `/var/log/nova/` in the loop

2) Calling single `add_copy_spec` means sizelimit is applied to *all* arguments altogether. Could you please replace it by something like:

    for _dir in self._subdirs('/var/log/containers'):
        self.add_copy_spec(_dir)

(or maybe have the `add_copy_spec` directly in `_subdirs` method? Or inline the method directly to `setup`? whatever shall be ok..)

Reply:
Didn't realize that was you broking my PR ;).

Lemme know if the update suits your needs. We're on the same TZ, we can sort this quick hopefully.